### PR TITLE
Update PX4 ROS2 telemetry section

### PR DESCRIPTION
**Pull request by GuillaumeLaine**
### Changes
- Update section on Telemetry in PX4 ROS2 interface lib docs

### To Do
- [x] Merge https://github.com/Auterion/px4-ros2-interface-lib/pull/31
- [x] Deploy doxygen docs for odometry attitude: https://github.com/Auterion/px4-ros2-interface-lib/pull/32

FYI @bkueng @bastianhjaeger

### CODE REVIEW

File: en/ros2/px4_ros2_control_interface.md [hunk 1]
```diff
@@ -419,10 +419,24 @@ If you want to control an independent actuator (a servo), follow these steps:
 
 ### Telemetry
 
-Telemetry topics include:
+You can access PX4 telemetry topics directly via the following classes:
 
 - [OdometryGlobalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryGlobalPosition.html): Global position
-- [OdometryLocalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryLocalPosition.html): Local position, velocity and acceleration.
+- [OdometryLocalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryLocalPosition.html): Local position, velocity, acceleration, and heading.
+- [OdometryAttitude](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryAttitude.html): Vehicle attitude
```
Comment (hamishwillee) [inline en/ros2/px4_ros2_control_interface.md:426]:
```suggestion
- [OdometryAttitude](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryAttitude.html): Vehicle attitude.
```

File: en/ros2/px4_ros2_control_interface.md [hunk 2]
```diff
@@ -419,10 +419,24 @@ If you want to control an independent actuator (a servo), follow these steps:
 
 ### Telemetry
 
-Telemetry topics include:
+You can access PX4 telemetry topics directly via the following classes:
 
 - [OdometryGlobalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryGlobalPosition.html): Global position
-- [OdometryLocalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryLocalPosition.html): Local position, velocity and acceleration.
+- [OdometryLocalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryLocalPosition.html): Local position, velocity, acceleration, and heading.
```
Comment (hamishwillee) [inline en/ros2/px4_ros2_control_interface.md:425]:
```suggestion
- [OdometryLocalPosition](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1OdometryLocalPosition.html): Local position, velocity, acceleration, and heading
```

**Review by hamishwillee — APPROVED** [review]
Great job @GuillaumeLaine 

As an aside, if you [look here](https://auterion.github.io/px4-ros2-interface-lib/annotated.html) some of the doxygen annotation appears to be missing (i.e. an easy improvement to docs to start with a very simple high level summary of purpose).

image

**Comment by bastianhjaeger** [comment]
@GuillaumeLaine Great! Thx!

**Pull request merged**